bbc/simorgh
# WSTEAM1-355: Create media promo list and promo list item component

Resolves JIRA WSTEAM1-355 and WSTEAM1-353 (subtasks of WSTEAM1-307)

Overall changes
======
Create a new 'Latest Media' component to show on Media Article Pages.

Code changes
======
- Create a `LatestMediaSection` within `MediaArticlePage` > `PagePromoSections`.
- Update the MediaArticlePage SecondaryColumn file to show the Latest Media component.
- Add Fixture data for Media Article Pages (Hausa, Pidgin, Tamil)
- Add unit tests
- Add stories for the LatestMediaSection and update Media Article Page stories.

Outstanding
======
- Remove Top Stories from `MediaArticlePage` > `PagePromoSections`. We will do this once the PR is code reviewed.

Testing
======
- Pull down branch and launch the local server
- Visit the following links to to see the new component
- http://localhost:7080/hausa/articles/cxr0765kxlzo
- http://localhost:7080/tamil/articles/c84m2jl4dpzo
- http://localhost:7080/pidgin/articles/cw0x29n2pvqo

To view without firing up a local dev server, use our Storybook stories:
- https://wsteam1-355-media-promo-list-componen--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&id=pages-media-article-page--media-article-page&viewMode=story
- https://wsteam1-355-media-promo-list-componen--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&id=pages-media-article-page--media-article-page-with-latest-media-images&viewMode=story
- https://wsteam1-355-media-promo-list-componen--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&id=pages-media-article-page--media-article-page-with-single-latest-media&viewMode=story

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
